manuscripts2/report.py

```diff
@@ -81,69 +90,185 @@ def __init__(self, es_url=None, start=None, end=None, data_dir=None, filters=Non
         self.data_dir = data_dir
         self.index_dict = defaultdict(lambda: None)
         for pos, index in enumerate(indices):
-            self.index_dict[data_sources[pos]] = Index(index_name=index)
+            self.index_dict[data_sources[pos]] = index
+
+        self.data_sources = data_sources
 
-        self.config = self.__get_config(data_sources=data_sources)
+        self.start_date = datetime(2015, 1, 1)
+        self.end_date = datetime(2018, 7, 10)
+
+        # self.config = self.__get_config(data_sources=data_sources)
 
     def get_metric_index(self, data_source):
```

Comment:
Can you include text to define this function? I'm not sure what it is intended to do, exactly.

Reply:
This function is supposed to return the elasticsearch index for a corresponding data source. As the user can pass their own elasticsearch index names for each of the data sources, this function chooses in between the default and the user inputed es indices and returns the user inputed one if it is available

Reply:
I see. Please, add that as the description of the function.